js/menu.js

```diff
@@ -34,12 +34,12 @@ function init() {
   $('.fl-menu-overlay').click(function() {
     $(this).closest('.fl-menu').removeClass('active');
     $('.fl-viewport-header .hamburger').removeClass('is-active');
-    $('body').removeClass('has-slide-menu');
+    $('body, body .container-fluid').removeClass('has-slide-menu');
```

Comment:
Why does `body .container-fluid` need to be changed too? `.container-fluid` is a very generic class used in Bootstrap. We don't want to be modifying the overflow, position and height properties for all the ones that appear on the page.

Reply:
@tonytlwu I've added this class to 'body .container-fluid' because safari browsers do not support overflow hidden for body tag and it need just only for opened menu to prevent scrolling content under menu for iOS mobile app.

Reply:
I see. `body .container-fluid` shouldn't have its styles manipulated as the selector is too general across the Bootstrap framework. You need to figure out another way.

Reply:
@tonytlwu Please check my last changes. Are you happy with that.

Reply:
@maksim-an-upplabs Am I right that your code involves taking...

```html
<body>
  <!-- Content -->
</body>
```

...and changing the structure to become...

```html
<body>
  <div class="scroll-wrapper">
    <!-- Content -->
  </div>
</body>
```

If so, we can't do that. We can be changing the structure of HTML content, and definitely not just because the user has chosen to use a specific menu. Users could be writing CSS and JS code that relied on a set HTML structure.